AllReadyApp/Web-App/AllReady/Areas/Admin/Controllers/SiteAdminController.cs

```diff
@@ -61,7 +61,9 @@ public IActionResult Index()
         [ValidateAntiForgeryToken]
         public async Task<IActionResult> ConfirmDeleteUser(string userId)
         {
-            await _mediator.SendAsync(new DeleteUserCommand { UserId = userId });
+            var user = await _userManager.FindByIdAsync(userId);
+            await _userManager.DeleteAsync(user);

```

Comment:
Actually, the PR uses the injected `UserManager<ApplicationUser>` to delete the user, so we would not need to inject any other dependencies.

If we discover going forward we need to perform this operation from multiple places, we can always revert to the handler doing the deletion taking `UserManager<ApplicationUser>` as a dependency. 

Not over thinking it, it's a good convo to have. Thanks for taking the time to explain.